core/src/stores/postgres.rs

```diff
@@ -1879,3 +1890,22 @@ impl Store for PostgresStore {
         Box::new(self.clone())
     }
 }
+
+async fn get_data_source_row_id(
```

Comment:
Let's not dry this one either for now. All methods have full implementation with queries visible there. Let's keep it that way unless we decide to explicitely dry stuff out.

Reply:
:ok: